test/recipes/90-test_store.t

```diff
@@ -10,10 +10,22 @@ use File::Spec;
 use File::Copy;
 use MIME::Base64;
 use OpenSSL::Test qw(:DEFAULT srctop_file srctop_dir bldtop_file data_file);
+use OpenSSL::Test::Utils;
 
 my $test_name = "test_store";
 setup($test_name);
 
+# Programs built with mingw have init code that mangles arguments that look
+# like paths.  The exact conversion rules are listed here:
+#
+#       http://www.mingw.org/wiki/Posix_path_conversion
+#
+# It's possible to avoid conversion by defining MSYS2_ARG_CONV_EXCL with
+# some suitable pattern ("*" to avoid conversions entirely), but that has
+# other unsuitable side effects on paths in the form of URIs, so best avoid
+# them on mingw.
+my $mingw = config('target') =~ m|^mingw|;
```

Comment:
Note that this test does pass with mingw cross-compile on Linux. So it's not really `config('target')` that is the problem, but `$^O`.

Reply:
Ah!  I forgot that perl on msys had its own platform name...  Okie, will fix

Reply:
> `$^O`

Well, it's combination really. If you were to build openssl with MSYS run-time, the test would pass.